mnemonic-memory-services/mnemonic-pmalloc-service/src/main/native/CMakeLists.txt

```diff
@@ -40,12 +40,29 @@ include_directories(${JNI_INCLUDE_DIRS})
 find_package(Threads REQUIRED)
 include_directories(${CMAKE_THREAD_LIBS_INIT})
 
-find_library(LIBPMALLOC_LIBRARIES NAMES libpmalloc_static.a)
-if (NOT LIBPMALLOC_LIBRARIES)
-   message(FATAL_ERROR "not found pmalloc library")
-endif (NOT LIBPMALLOC_LIBRARIES)
+#find_library(LIBPMALLOC_LIBRARIES NAMES libpmalloc_static.a)
```

Comment:
It can be removed if don't need it anymore, it is okay to keep it for reference as well, thanks.